csharp/ql/examples/qlpack.yml

```diff
@@ -1,3 +1,4 @@
 name: codeql-csharp-examples
 version: 0.0.0
-libraryPathDependencies: codeql-csharp
+dependencies:
+  codeql/csharp-all: 0.0.1
```

Comment:
Should this be 0.0.2, or ^0.0.1?

Reply:
Good catch. That should be made more flexible.